### PR TITLE
RB-586: Remove APIKey being passed to chat UI

### PR DESCRIPTION
**Pull request by LawrieR**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ facing release notes.
   *  [Fix] A bug fix
   * [Misc] Other items
 
+## v0.0.13
+
+  *  [Fix] RB-586: Remove API key from front end.
+
 ## v0.0.1
```
Comment (LawrieR) [inline CHANGELOG.md:18]: Embarrassingly I forgot to update the changelog for the last 12 versions.

File: public/chirpy.js [hunk 2]
```diff
@@ -25,32 +26,30 @@ function start () {
...
-                        });
+                                clearUserInput();
+                            });
+                        } else {
+                            var ourQuery = {
+                                subject: goal.subjectInstance ? goal.subjectInstance : null,
+                                relationship: goal.rel,
+                                object: goal.objectInstance ? goal.objectInstance : null
+                            };
+                            rapi.query(ourQuery, handleResponse);
```
Comment (craig-rainbird) [inline public/chirpy.js:51]: Could this be done just once below the if statement? ourQuery declared just before if statement with goal.rel being common between the two?

File: public/chirpy.js [hunk 3]
```diff
@@ -13,6 +13,7 @@ function start () {
             console.error(error, status);
         } else {
             console.log(agent);
```
Comment (craig-rainbird) [inline public/chirpy.js:15]: I think it might be for EC or something, but is this required or is it just useful?

Comment (LawrieR) [reply public/chirpy.js:15]: Not required. The other agent prints out some similar info so I thought I'd keep it about in this version too.

**Pull request merged**